### DISCUSSION TITLE
Destination host unreachable

### DISCUSSION THREAD

**Discussion by dgaz9** [Security-Onion-Solutions/securityonion#5219] [Unsupported Versions]
Hello all! I have just set up SO and unable to connect.its not the router, or my switch. The interface is showing as up without issues but no actual connections are made. When I ping anything including gateway I only get destination host unreachable. Any help is appreciated!

**Comment by gcsrellis** [comment]
There's a lot of variables that could cause this.  Is this virtualized with ESXi or Workstation or some other Virtual software?  Do you have the port configured correctly for the VLAN?  You need to troubleshoot some of these basic things first.  Security Onion requires a decent level of base knowledge and experience to use.
